packages/table/src/table.tsx

```diff
@@ -425,6 +422,10 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         selectionModes: SelectionModes.ALL,
     };
 
+    public static childContextTypes: React.ValidationMap<
+        IColumnInteractionBarContextTypes
+    > = columnInteractionBarContextTypes;
```

Comment:
`public static childContextTypes = columnInteractionBarContextTypes` should do it.

Reply:
nope, see above